driver-core/src/main/com/mongodb/MongoClientSettings.java

```diff
@@ -297,6 +297,8 @@ public Builder writeConcern(final WriteConcern writeConcern) {
         /**
          * Sets whether writes should be retried if they fail due to a network error.
          *
+         * <p>Starting with the 3.11.0 release, the default value is true</p>
```

Comment:
Seems appropriate to call out when the default changed (since we rarely change defaults like this), but let me know if you think it's over the top.

Reply:
I would agree that this is an appropriate statement.

Reply:
:+1: